pom.xml

```diff
@@ -73,19 +86,17 @@
       <version>2.1.5</version>
     </dependency>
   </dependencies>
-
-  <!-- get every artifact through repo.jenkins-ci.org, which proxies all the artifacts that we need -->
```

Comment:
Is there a big reason to remove this comment? Or was it cut as "a" comment along with other cleanup?

Reply:
It's because it's a "what" comment and not a "why" comment. 
Also it's kind of obvious while on the other hand not being technically correct (it's not a proxy)

Reply:
Ok, thanks. Guess a "mirror" would be more correct :)

Reply:
Yes, FYI: another thing is it's just adding a new repository to the list, it doesn't "disable" maven central etc.